site/content/docs/main/namespace.md

```diff
@@ -19,4 +19,8 @@ velero client config set namespace=<NAMESPACE_VALUE>
 
 Alternatively, you may use the global `--namespace` flag with any operational command to tell Velero where to run.
 
+## Customize the targeted namespace
+
+In the YAML deployment file using Velero's image, define the environment variable NAMESPACE_NAME to specify the namespace in which you want Velero to backup resources.
```

Comment:
I'm not sure this documentation is accurate. From what I'm reading, the environment variable limits what pods restic can see. The way I read this note implies this is the only way to target a namespace is via this environment variable.

The user still has to select a relevant namespace with the `--include-namespaces` and `--exclude-namespaces` arguments, unless I'm misunderstanding your deployment setup.

Reply:
+1 This is restricting the pods that restic can see and that is probably not what you want.
@sfelix-orange can you please share what is the problem you are trying to solve? We can perhaps propose a solution to that.